packages/polyserve/src/test/cli_test.ts

```diff
@@ -58,6 +58,8 @@ suite('cli', () => {
       }
     } finally {
       unintercept();
+      console.log('stdout from failed cli invocation:');
+      console.log(stdout);
```

Comment:
Are these leftover as well?

Reply:
This are intended, the other ones aren't